libs/ledgerjs/packages/cryptoassets/src/types.ts

```diff
@@ -1,5 +1,7 @@
+import { CryptoCurrencyId } from "@ledgerhq/types-cryptoassets";
+
 export type ERC20Token = [
```

Comment:
While we're here, any reason for not moving this type to `types-cryptoassets`?

Reply:
No reason I just didn't want to add changes not related to AVAX inte